src/test/java/pl/wrapper/parking/exception/ResultTest.java

```diff
@@ -6,7 +6,6 @@
 import pl.wrapper.parking.result.SuccessResult;
 import static org.junit.jupiter.api.Assertions.*;
 
-
 class ResultTest {
```

Comment:
Test what happens when .block() get's called on Mono.error in PwrApiServerCallerImpl.fetchData()